File: app/src/main/java/com/acceleratorer/wuwavn/PatchManifestRepository.java
package com.acceleratorer.wuwavn;

final class PatchManifestRepository {
    PatchManifest current() {
        return new PatchManifest(
                "2026.05.15",
                "https://github.com/CallMeDangDev/WuwaVH/releases/latest/download/WuWaVH_99_P.pak",
                "99202f84db73e771cd68ef1734893ee5f589666654fe7be3dcfff631df199a78"
        );
    }
}

File: app/src/main/kotlin/com/acceleratorer/wuwavn/PatchManifestRepository.kt
package com.acceleratorer.wuwavn

class PatchManifestRepository {
    fun current(): PatchManifest = PatchManifest(
        patchVersion = "wuwa-3.3.6-vi-2026.05",
        pakUrl = "https://github.com/CallMeDangDev/WuwaVH/releases/download/3.3.6/WuWaVH_99_P.pak",
        pakSha256 = "146abe4d4d0d17036776ac82ba26c0b4c6b1a4f073b8c84fed91a327d013eb85",
        pakFileName = "WuWaVH_99_P.pak",
    )
}
